packager/packager.cc

```diff
@@ -837,7 +844,8 @@ bool checkContainsAllPackages(const core::GlobalState &gs, const vector<ast::Par
 
 } // namespace
 
-vector<ast::ParsedFile> Packager::run(core::GlobalState &gs, WorkerPool &workers, vector<ast::ParsedFile> files) {
+vector<ast::ParsedFile> Packager::run(core::GlobalState &gs, WorkerPool &workers, vector<ast::ParsedFile> files,
+                                      vector<std::string> extraPackageFilesDirectoryPrefixes) {
```

Comment:
Instead of copying `extraPackageFilesDirectoryPrefixes` on a call to this function, can we pass it as `const &`?